airbyte-webapp/src/components/Modal/ModalBody.module.scss

```diff
@@ -2,6 +2,6 @@
 
 .modalBody {
   padding: variables.$spacing-lg variables.$spacing-xl;
-  overflow: scroll;
+  overflow: auto;
```

Comment:
:information_source: @edmundito I've changed this to `auto`, since `scroll` will have scrollbars always visible (on non mac systems) inside the modal body:

![screenshot-20220715-111421](https://user-images.githubusercontent.com/877229/179193344-67fa56b1-2879-4b75-af4d-5799af1e5e66.png)

Reply:
Whoops, yes I am familiar with this. it definitely should be auto!